core/imgread/common.cpp

```diff
@@ -131,7 +131,7 @@ bool ConvertSector(u8* in_buff , u8* out_buff , int from , int to,int sector)
 
 Disc* OpenDisc(const wchar* fn)
 {
-	Disc* rv = nullptr;
+	Disc* rv = 0;
```

Comment:
This should be fixed in a separate PR, not in a "documentation fixes" PR.

Reply:
The two PRs' commits have been mixed up.
This is originally included in the Snapcraft fixes PR.
Let me fix this real quick and review again.

Reply:
Yes, please move this commit to #1430 as that would be where it belongs.

Reply:
It looks loke you're no longer working on this? There are more issues regarding this single problem.
I might reopen my fix to address this single problem.

Reply:
This is no longer necessary as it's fixed in master.